src/bundle/Resources/config/universal_discovery_widget.yml

```diff
@@ -37,3 +37,6 @@ system:
                     multiple: false
                     content_on_the_fly:
                         allowed_content_types: ['image']
+                image_asset:
+                    multiple: false
+                    visible_tabs: ['browse', 'search', 'bookmarks']
```

Comment:
Why you should not be able to create content while picking from UDW?

Reply:
Good question. I going to check this with PM team 😉